flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.html

```diff
@@ -22,6 +22,12 @@
             <text class="Unit" x="150" y="87" text-anchor="start">&#176;C</text>
         </g>
 
+        <g id="ISA">
+            <text class="Title" x="74" y="111" text-anchor="end">ISA</text>
+            <text id="ISAValue" x="112" y="111" text-anchor="middle" class="Value">XX</text>
+            <text class="Unit" x="150" y="111" text-anchor="start">&#176;C</text>
+        </g>
```

Comment:
Hi! When referring to your images, the aligning is a bit wrong (also for the other ones).

These should be right aligned and not centered as your single digit shows.

Try to set the `text-anchor` to `end`, and then adjust the `x`-axis accordingly, for **all** these 3 svg groups.

Reference: 
![isa](https://user-images.githubusercontent.com/433429/121441389-fbf87a00-c989-11eb-8e6c-382571153f2a.png)

Mock/p.o.c in actual svg:
![isa-svg](https://user-images.githubusercontent.com/433429/121441619-73c6a480-c98a-11eb-9305-33e79bda6698.png)

Br.

/J